res/xml/firmware_version.xml

```diff
@@ -62,7 +62,11 @@
         android:key="maintainer"
         android:title="@string/maintainer"
         android:summary="@string/maintainer_name"
-        settings:enableCopying="true" />
+        settings:enableCopying="true" >
+        <intent
+                android:action="android.intent.action.VIEW"
+                android:data="@string/maintainer_telegram"/>
```

Comment:
Should add `maintainer_telegram` default to strings xml. Link it to alex's TG

Reply:
But it's for device maintainer, overlay can be set alongwith "maintainer_name" string.
Like..
    <!-- DerpFest maintainer -->
	<string name="maintainer_name">Abuzar Shaheen(Ab.Shaheen)</string>
	<string name="maintainer_telegram">https://t.me/AbuzarShaheen</string>